lib/MusicBrainz/Server/Data/ReleaseGroup.pm

```diff
@@ -86,6 +86,11 @@ sub _where_filter
             push @query, "rg.type = ?";
             push @params, $filter->{type_id};
         }
+        if (exists $filter->{secondary_type_id}) {
+            push @query, 'st.secondary_type = ?';
+            push @params, $filter->{secondary_type_id};
+            push @joins, 'JOIN release_group_secondary_type_join st ON rg.id = st.release_group';
+        }
         if (exists $filter->{type} && $filter->{type}) {
```

Comment:
I'm not sure this conditional does anything at all ever?

Reply:
It's used in the web-service, but I don't think anywhere else: https://github.com/metabrainz/musicbrainz-server/blob/c246080/lib/MusicBrainz/Server/Controller/WS/2/Artist.pm#L103-L104